src/test/java/org/opensearch/sdk/TestSDKClient.java

```diff
@@ -10,52 +10,81 @@
 
         sdkClient.doCloseJavaClient();
     }
 
     @Test
-    public void testCreateHighLevelRestClient() throws Exception {
-        RestHighLevelClient testClient = sdkClient.initializeRestClient("localhost", 9200);
+    public void testCreateRestClient() throws Exception {
+        SDKRestClient restClient = sdkClient.initializeRestClient("localhost", 9200);
+        assertInstanceOf(SDKIndicesClient.class, restClient.indices());
+        assertInstanceOf(SDKClusterAdminClient.class, restClient.cluster());
 
-        // Using the package name here as Java uses package name if the filename from different packages are same
-        org.opensearch.client.indices.CreateIndexRequest createIndexRequest = new org.opensearch.client.indices.CreateIndexRequest(
-            "my-index"
-        );
+        sdkClient.doCloseHighLevelClient();
+    }
+
+    @Test
+    public void testSDKRestClient() throws Exception {
+        SDKRestClient restClient = sdkClient.initializeRestClient("localhost", 9200);
 
-        assertThrows(ConnectException.class, () -> testClient.indices().create(createIndexRequest, RequestOptions.DEFAULT));
+        // Would really prefer to mock/verify the method calls but they are final
+        assertDoesNotThrow(() -> restClient.index(new IndexRequest(), ActionListener.wrap(r -> {}, e -> {})));
+        assertDoesNotThrow(() -> restClient.get(new GetRequest(), ActionListener.wrap(r -> {}, e -> {})));
+        assertDoesNotThrow(() -> restClient.delete(new DeleteRequest(), ActionListener.wrap(r -> {}, e -> {})));
+        assertDoesNotThrow(() -> restClient.search(new SearchRequest(), ActionListener.wrap(r -> {}, e -> {})));
 
-        sdkClient.doCloseHighLevelClient();
+        restClient.close();
+    }
+
+    @Test
+    public void testSDKIndicesClient() throws Exception {
+        SDKRestClient restClient = sdkClient.initializeRestClient("localhost", 9200);
+        SDKIndicesClient indicesClient = restClient.indices();
+
+        // Would really prefer to mock/verify the method calls but the IndicesClient class is final
+        assertInstanceOf(Cancellable.class, indicesClient.create(new CreateIndexRequest(""), ActionListener.wrap(r -> {}, e -> {})));
+        assertInstanceOf(Cancellable.class, indicesClient.delete(new DeleteIndexRequest(), ActionListener.wrap(r -> {}, e -> {})));
+        assertInstanceOf(Cancellable.class, indicesClient.getMapping(new GetMappingsRequest(), ActionListener.wrap(r -> {}, e -> {})));
+        assertInstanceOf(Cancellable.class, indicesClient.putMapping(new PutMappingRequest(), ActionListener.wrap(r -> {}, e -> {})));
+        assertInstanceOf(
+            Cancellable.class,
+            indicesClient.rolloverIndex(new RolloverRequest("", ""), ActionListener.wrap(r -> {}, e -> {}))
+        );
+        assertInstanceOf(Cancellable.class, indicesClient.getAliases(new GetAliasesRequest(), ActionListener.wrap(r -> {}, e -> {})));
 
+        restClient.close();
```

Comment:
We need to close sdkClient here too. 
```
 sdkClient.doCloseHighLevelClient()

Reply:
All this does is call `restClient.close()` with a null check, which accomplishes the same thing here.  The only difference would be test coverage, but that's covered at line 55.

I'll go ahead and change all the individual client close calls to  this model, though.